<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="txtCampoTexto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>691</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardarArchivo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/save.png</normaloff>../../Downloads/save.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>761</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBorrarTexto">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/eraser.png</normaloff>../../Downloads/eraser.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrivado">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/padlock.png</normaloff>../../Downloads/padlock.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
